domain/src/main/java/edu/nextstep/camp/calculator/domain/validator/CalculatorInputValidator.kt

```diff
@@ -0,0 +1,41 @@
+package edu.nextstep.camp.calculator.domain.validator
+
+import edu.nextstep.camp.calculator.domain.util.CalculatorInputUtil
+
+class CalculatorInputValidator {
+    companion object {
```

Comment:
```suggestion
object CalculatorInputValidator {
```
클래스 내부에 companion object만 있는 경우에는, class 대신 object를 활용해주시는 것이 좋습니다.